src/transformers/__init__.py

```diff
@@ -3569,6 +3583,12 @@
         TransfoXLCorpus,
         TransfoXLTokenizer,
     )
+    from .models.trillsson_efficient import (
+        TRILLSSON_EFFICIENT_PRETRAINED_CONFIG_ARCHIVE_MAP,
+        Trillsson_efficientConfig,
+        Trillsson_efficientFeatureExtractor,
```

Comment:
Should be imported [here](https://github.com/huggingface/transformers/blob/main/src/transformers/__init__.py#L3885)